src/d_main.c

```diff
@@ -2104,6 +2104,13 @@ void D_DoomMain(void)
 
   if (beta_emulation)
     {
+      char *path = D_FindWADByName("betagrph.wad");
+      if (path == NULL)
+      {
+        I_Error("'BETAGRPH.WAD' is required for beta emulation! "
```

Comment:
Is it really necessary to error out here? Can't beta emulation work without `betagrph.wad` or can we work around its missing assets? For example, we already have the beta plasma balls hard-coded into the engine.

Reply:
I think that without the beta graphics, it would be difficult to distinguish it from the usual gameplay.